authors/apps/articles/serializers.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import serializers
+from .models import Article
+
+
+class ArticleSerializer(serializers.ModelSerializer):
+    """Serializer to map the Article Model instance into JSON format."""
+
+    title = serializers.CharField(max_length=100)
+    description = serializers.CharField(max_length=128)
+    body = serializers.CharField()
+    # author = serializers.IntegerField()
+
+    def create(self, validated_data):
+        return Article.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.title = validated_data.get('title', instance.title)
+        instance.description = validated_data.get(
+            'description', instance.description)
+        instance.body = validated_data.get('body', instance.body)
+        # instance.author_id = validated_data.get('author_id', instance.author_id)
```

Comment:
Same goes for this. kindly remove the commented out

Reply:
Also done, the feedback is appreciated.